src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -6,30 +6,94 @@
 
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 import com.ctre.phoenix.motorcontrol.can.TalonFX;
 
 import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
 
 public class Climber extends SubsystemBase {
-
   CANSparkMax m_elevatorMotorLeader;
   CANSparkMax m_elevatorMotorFollower;
-  TalonFX m_armMotorLeader;
-  TalonFX m_armMotorFollower;
+  CANSparkMax m_armMotorLeader;
+  CANSparkMax m_armMotorFollower;
   RelativeEncoder m_winchEncoder;
-  DutyCycleEncoder m_armEncoder;
+  RelativeEncoder m_armEncoder;
```

Comment:
We still want to keep the DutyCycleEncoder. That will be the Rev Through-Bore encoder. We'll use it to initialize the current arm angle and also to set limits on how far the arm can go. Note that the shaft it is on has a 16 tooth sprocket. The chain is connected to a 60 tooth sprocket on the arm. So we'll need to account for that in converting the absolute encoder reading to the current arm angle.